pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -442,9 +443,9 @@ class ClassTypeHintMatch:
         ),
     ],
 }
-# Properties are normally checked by mypy, and will only be checked
-# by pylint when --ignore-missing-annotations is False
-_PROPERTY_MATCH: dict[str, list[ClassTypeHintMatch]] = {
+# Overriding properties and functions are normally checked by mypy, and will only
```

Comment:
```suggestion
# Overridden properties and functions are normally checked by mypy, and will only
```
?

Reply:
That was my original thought, but I amended with "Overriding" because those properties and functions are overriding base methods. They are not overridden by a child class...

Reply:
Ok, let's leave it. It's just a comment after all.

Would like like to add a comment for the conflict with `arg_types` and `kwargs_type` I mentioned?